config/bolt/contenttypes.yaml

```diff
@@ -64,13 +64,16 @@ pages:
             type: text
             class: large
             group: content
+        subtitle:
+            type: text
         slug:
             type: slug
             uses: title
         image:
             type: image
         content:
             type: article
+            allow_twig: true

```

Comment:
Hi @mcdennem , I don't think we use `allow_twig` just yet. This is likely something left from when we worked together :-) Let's get rid of it.

Reply:
Check